### PR TITLE
Update participate.md

### PR DESCRIPTION
**Pull request by jsubida**
## Description

Add content from [Google Doc](https://docs.google.com/document/d/1T1LkjQ-Ug38aPCo7gkV3tW5UOy4IJWor1ntp6LJxSGQ/edit?usp=sharing)

## Checklist:

- [ ] (Committee Chairs Only) I have posted the link for the PR in the usrse slack (#usrse-2024-conference-planning-committee) to ask for content reviewers
- [x] I have previewed changes locally
- [ ] I have updated the README.md if necessary

cc  @jrgochan

### CODE REVIEW

File: pages/participate/participate.md [hunk 1]
```diff
@@ -4,8 +4,61 @@ title: Participate
...
+
+Accepted abstracts will be placed on the website ahead of the conference.
+
+## Important Dates
+
+- **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
+- **BoFs, Workshops, and Tutorials**: Due March 22. Notifications April 26.
+- **Papers and Notebooks**: Due May 20. Notifications June 27.
+- **Posters and Talks**: Due June 3. Notifications July 3.
+- **Camera-ready submissions (Notebooks, Papers)**: August 9
```
Comment (mrmundt) [inline pages/participate/participate.md:36]: Please add days of the week (e.g., "Friday, March 22"). The extra clarity never hurts!

File: pages/participate/participate.md [hunk 2]
```diff
@@ -4,8 +4,61 @@ title: Participate
...
+
+## Birds of a Feather (BoFs)
+
+Birds of a Feather sessions are expected to be informal, and ideally will include the audience’s participation. Sessions will last 90 minutes and take place in rooms that will accommodate approximately 100 people. Organizers may choose to submit with all presenters determined at the time of submission, or include a plan for finding BoF presenters upon acceptance. Submission template [here](https://docs.google.com/document/d/1W86USaI0euZOuwPJoeZfZaX-qxG_OjRTM3brHNxNIvM/edit?usp=sharing).
```
Comment (mrmundt) [inline pages/participate/participate.md:40]: Unnecessary comma
```suggestion
Birds of a Feather sessions are expected to be informal, and ideally will include the audience’s participation. Sessions will last 90 minutes and take place in rooms that will accommodate approximately 100 people. Organizers may choose to submit with all presenters determined at the time of submission or include a plan for finding BoF presenters upon acceptance. Submission template [here](https://docs.google.com/document/d/1W86USaI0euZOuwPJoeZfZaX-qxG_OjRTM3brHNxNIvM/edit?usp=sharing).
```

File: pages/participate/participate.md [hunk 3]
```diff
@@ -4,8 +4,61 @@ title: Participate
...
+
+## Tutorials
+
+Tutorials will take place ahead of the conference in a virtual format, and be recorded, from September 30th to October 11th. Infrastructure to register participants and allow the hosts to run the tutorial via Zoom will be provided. Tutorials may be either 90 minutes or 2 90 minute sessions with a 30 minute break in the middle. Please note that you ***MAY*** submit similar proposals for both a tutorial and a Learning Topic workshop if you would like to deliver the same content in person at the conference. Submission template [here](https://docs.google.com/document/d/1RljR4M2kEBZx36JmSNLrw0iL7P-cx3aoosYdnHJR8t4/edit?usp=sharing).
```
Comment (mrmundt) [inline pages/participate/participate.md:48]:
```suggestion
Tutorials will take place ahead of the conference in a virtual format (to be recorded) from September 30th to October 11th. Infrastructure to register participants and allow the hosts to run the tutorial via Zoom will be provided. Tutorials may be either 90 minutes or two 90 minute sessions with a 30 minute break in the middle. Please note that you ***MAY*** submit similar proposals for both a tutorial and a Learning Topic workshop if you would like to deliver the same content in person at the conference. Submission template [here](https://docs.google.com/document/d/1RljR4M2kEBZx36JmSNLrw0iL7P-cx3aoosYdnHJR8t4/edit?usp=sharing).
```

**Review by mrmundt — CHANGES_REQUESTED** [review]
Can you please also make an "Important Dates" page under `About`? I left an example from last year in the repo: https://github.com/USRSE/usrse24/blob/main/pages/about/dates.tbd

File: pages/participate/participate.md [hunk 4]
```diff
@@ -4,8 +4,61 @@ title: Participate
...
+- Diversity, Equity, and Inclusion for RSEs and in RSEng
+- Training and workforce development
+- Maturing and expanding the RSE profession
+
+Accepted abstracts will be placed on the website ahead of the conference.
+
+## Important Dates
+
+- **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
+- **BoFs, Workshops, and Tutorials**: Due March 22. Notifications April 26.
```
Comment (jbteves-sandia) [inline pages/participate/participate.md:33]:
```suggestion
- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
```

File: pages/participate/participate.md [hunk 5]
```diff
@@ -4,8 +4,61 @@ title: Participate
...
+- **Papers and Notebooks**: Due May 20. Notifications June 27.
```
Comment (jbteves-sandia) [inline pages/participate/participate.md:34]:
```suggestion
- **Papers and Notebooks**: Due Monday, May 20. Notifications Friday, June 28.
```
Note that I messed that up, I intended for it to be Friday.

File: pages/participate/participate.md [hunk 6]
```diff
@@ -4,8 +4,61 @@ title: Participate
...
+- **Posters and Talks**: Due June 3. Notifications July 3.
```
Comment (jbteves-sandia) [inline pages/participate/participate.md:35]:
```suggestion
- **Posters and Talks**: Due Monday, June 3. Notifications Wednesday, July 3.
```

File: pages/participate/participate.md (hunk 1 above)
Comment (jbteves-sandia) [inline pages/participate/participate.md:36]:
```suggestion
- **Camera-ready submissions (Notebooks, Papers)**: Friday, August 9
```

**Comment by mrmundt** [comment]
FYI - there was an incorrect setting on CircleCI that was preventing building changes from forks. I have changed that so all PRs _should_ get CircleCI previews now.

File: pages/about/dates.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- **Submissions open**: Monday, February 12 for all but notebooks. Notebooks to open Monday, February 26.
+- **BoFs, Workshops, and Tutorials**: Due Friday, March 22. Notifications Friday, April 26.
+- **Papers and Notebooks**: Due Monday, May 20. Notifications Friday, June 28.
+- **Posters and Talks**: Due Monday, June 3. Notifications Wednesday, July 3.
+- **Camera-ready submissions (Notebooks, Papers)**: Friday, August 9
+
+## Conference
+
+- **Main Conference**: October 15-17, 2023, Albuquerque, NM
```
Comment (mrmundt) [inline pages/about/dates.md:20]:
```suggestion
- **Main Conference**: October 15-17, 2024, Albuquerque, NM
```

**Comment by jbteves-sandia** [comment]
Yes, please change to "view with link."

**Comment by jsubida** [comment]
> Yes, please change to "view with link."

@jbteves-sandia @mrmundt Should I change `templates.tbd` to `templates.md` and update `navigation.yml`? 

https://github.com/USRSE/usrse24/blob/main/pages/participate/templates.tbd

**Comment by mrmundt** [comment]
> > Yes, please change to "view with link."
> 
> @jbteves-sandia @mrmundt Should I change `templates.tbd` to `templates.md` and update `navigation.yml`?
> 
> https://github.com/USRSE/usrse24/blob/main/pages/participate/templates.tbd

No, not this time. The templates are linked in the call itself. It'll be less work to _not_ have a different page that repeats the same information as the Google docs.

**Review by mrmundt — APPROVED** [review]
Thank you!

**Pull request merged**